src/syscall/syscall_libc_wasi.go

```diff
@@ -296,6 +296,10 @@ func Pipe2(p []int, flags int) (err error) {
 	return ENOSYS // TODO
 }
 
+func Getpagesize() int {
+	return 4096 // TODO
```

Comment:
This is 64kB in Go, which also makes a lot more sense to me (it matches the wasm page size): https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/runtime/os_js.go;l=105

Reply:
Done.